Pandemic/Cards.swift

```diff
@@ -8,14 +8,6 @@
 
 import Foundation
 
-private extension Array {
```

Comment:
This is good. The built-in swap can be used instead. The only reason why I didn't use the built-in originally is that I couldn't get it to work. I expect I was calling it wrong. (Was I passing in references to the indexes rather than references to the actual cells? If so that would fail as both index vars are immutable)